### PR TITLE
Authorization + Graph.Rbac: update 2018 09 01

### PR DESCRIPTION
**Pull request by ChenTanyi**
https://github.com/Azure/azure-libraries-for-net/issues/522

### CODE REVIEW

File: src/ResourceManagement/Graph.RBAC/ActiveDirectoryGroupImpl.cs [hunk 1]
```diff
@@ -127,17 +127,17 @@ public IEnumerable<IActiveDirectoryObject> ListMembers()
...
         }
 
-        private IActiveDirectoryObject WrapObjectInner(AADObjectInner inner)
+        private IActiveDirectoryObject WrapObjectInner(DirectoryObject inner)
         {
-            var managementClient = ((GroupsOperations) manager.Inner.Groups).Client;
+            var managementClient = ((GroupsOperations)manager.Inner.Groups).Client;
             string serialized = SafeJsonConvert.SerializeObject(inner, managementClient.SerializationSettings);
-            switch (inner.ObjectType)
+            switch (inner.AdditionalProperties["objectType"])
```
Comment (yaohaizh) [inline src/ResourceManagement/Graph.RBAC/ActiveDirectoryGroupImpl.cs]: Should always avoid hard-coded string.

File: src/ResourceManagement/Graph.RBAC/PasswordCredentialImpl.cs [hunk 2]
```diff
@@ -123,7 +123,7 @@ internal PasswordCredentialImpl(Models.PasswordCredential passwordCredential)
         internal PasswordCredentialImpl(string name, IHasCredential<T> parent)
             : base(name, new Models.PasswordCredential()
             {
-                CustomKeyIdentifier = System.Convert.ToBase64String(System.Text.Encoding.UTF8.GetBytes(name)),
+                CustomKeyIdentifier = System.Text.Encoding.UTF8.GetBytes(name),
```
Comment (yaohaizh) [inline src/ResourceManagement/Graph.RBAC/PasswordCredentialImpl.cs]: Why change the encoding?

File: src/ResourceManagement/Graph.RBAC/ActiveDirectoryGroupImpl.cs (hunk 1 above)
Comment (ChenTanyi) [reply src/ResourceManagement/Graph.RBAC/ActiveDirectoryGroupImpl.cs]: this has been changed in third commit

File: src/ResourceManagement/Graph.RBAC/PasswordCredentialImpl.cs (hunk 2 above)
Comment (ChenTanyi) [reply src/ResourceManagement/Graph.RBAC/PasswordCredentialImpl.cs]: this generated field is changed from base64string to byte by swagger

**Pull request merged**